sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/sweep/sweep_job.py

```diff
@@ -148,7 +158,57 @@ def __init__(
         early_termination: Optional[Union[BanditPolicy, MedianStoppingPolicy, TruncationSelectionPolicy]] = None,
         queue_settings: Optional[QueueSettings] = None,
         **kwargs: Any,
-    ):
+    ) -> None:
+        """Sweep job for hyperparameter tuning.
+
+        :param name: Name of the job.
+        :type name: str
+        :param display_name: Display name of the job.
+        :type display_name: str
+        :param description: Description of the job.
+        :type description: str
+        :param tags: Tag dictionary. Tags can be added, removed, and updated.
```

Comment:
The second sentence implies that this can be used for somewhat complex behavior. Does the tags argument simply fully replace existing tags when non-empty, or do something more complex?

Reply:
The tags param comes from the base Resource class that almost all entities inherit. If you assign a dictionary as the tags param, it will override the existing one. If you try to update it like a dictionary, it will update. I'll find some language to fix here, but it will be repo-wide since this param and description is used everywhere.